plugins/modules/packaging/os/snap.py

```diff
@@ -105,151 +105,184 @@
     returned: When any snaps have been removed
 '''
 
-import operator
 import re
 
-from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.general.plugins.module_utils.module_helper import (
```

Comment:
I'm not quite sure about adding a dependency for an already working module on another module_utils from c.g. when it's working properly right now.

Reply:
Since the dependency is collection-internal, it is a question of whether this improves code quality (reduces repeated code, simplifies/improves code in general, ...) or not.

Reply:
@aminvakil I have been pulling some modules to use `ModuleHelper` instead of the plain `AnsibleModule`, but there is a number of those already like that, I should fo us on improving MH for a while instead of moving changing more modules.

Reply:
The whole purpose of MH is that of improving code quality.